$OpenBSD: patch-Makefile_am,v 1.1 2017/12/01 16:09:54 jca Exp $

Skip subdirectory which builds programs that need optional deps (sdl,
gtk), for which there is no easy --without-dep switch.

Index: Makefile.am
--- Makefile.am.orig
+++ Makefile.am
@@ -1,6 +1,6 @@
 ACLOCAL_AMFLAGS = -I m4
 
-SUBDIRS=libvncserver examples libvncclient webclients client_examples test
+SUBDIRS=libvncserver examples libvncclient webclients test
 DIST_SUBDIRS=libvncserver examples libvncclient webclients client_examples test
 EXTRA_DIST = CMakeLists.txt rfb/rfbint.h.cmake rfb/rfbconfig.h.cmake
 
